client/post-editor/editor-more-options/copy-post.jsx

```diff
@@ -58,54 +56,63 @@ class EditorMoreOptionsCopyPost extends Component {
 	}
 
 	goToNewDraft = () => {
-		if ( '' !== this.state.selectedPostId ) {
-			page.redirect( `/post/${ this.props.siteSlug }?copy=${ this.state.selectedPostId }` );
+		const { siteSlug, type } = this.props;
+		const { selectedPostId } = this.state;
+		if ( '' !== selectedPostId ) {
+			page.redirect( `/${ type }/${ siteSlug }?copy=${ selectedPostId }` );
 			this.closeDialog();
 		}
 	}
 
 	render() {
-		const { translate, siteId } = this.props;
+		const { siteId, translate, type } = this.props;
+		const { selectedPostId, showDialog } = this.state;
 		const buttons = [ {
 			action: 'cancel',
 			label: translate( 'Cancel' ),
 		}, {
 			action: 'copy',
 			label: translate( 'Overwrite' ),
 			isPrimary: true,
-			disabled: ! this.state.selectedPostId,
+			disabled: ! selectedPostId,
 			onClick: this.goToNewDraft,
 		} ];
 		return (
 			<AccordionSection className="editor-more-options__copy-post">
 				<EditorDrawerLabel
-					labelText={ translate( 'Copy Post' ) }
-					helpText={ translate( "Pick a post and we'll copy the title, content, tags and categories. " ) }
+					labelText={ this.isPost() ? translate( 'Copy Post' ) : translate( 'Copy Page' ) }
+					helpText={ this.isPost()
+						? translate( "Pick a post and we'll copy the title, content, tags and categories." )
```

Comment:
Yup, that's my bad.
I've added it with the trailing space in the Copy Post PR, and I thought it would make sense to fix it here.
If this is an issue for the i18n team, I can add the space back in no time.